Remotion/Data/DomainObjects/Queries/QueryFactory.cs

```diff
@@ -201,16 +235,17 @@ public static IQuery CreateQueryFromConfiguration (string id, QueryParameterColl
     /// <param name="storageProviderDefinition">The <see cref="StorageProviderDefinition"/> used to execute the query.</param>
     /// <param name="statement">The scalar query statement.</param>
     /// <param name="queryParameterCollection">The parameter collection to be used for the query.</param>
+    /// <param name="metaData">The metadata dictionary supplied to the query definition.</param>
```

Comment:
Since we are going to create the `QueryDefintion`, this should be: "The metadata dictionary that will be supplied to the `QueryDefintion`".